pkg/reporter/reporter.go

```diff
@@ -6,42 +6,47 @@ import (
 	"crypto/tls"
 	"encoding/json"
 	"fmt"
+	"io"
 	"net/http"
 	"net/url"
 	"time"
 
 	"github.com/anchore/ecs-inventory/internal/logger"
 	"github.com/anchore/ecs-inventory/internal/tracker"
 	"github.com/anchore/ecs-inventory/pkg/connection"
+	"github.com/h2non/gock"
```

Comment:
To fix the go imports error change imports to this:

```

import (
    "bytes"
    "crypto/tls"
    "encoding/json"
    "fmt"
    "io"
    "net/http"
    "net/url"
    "time"

    "github.com/h2non/gock"

    "github.com/anchore/ecs-inventory/internal/logger"
    "github.com/anchore/ecs-inventory/internal/tracker"
    "github.com/anchore/ecs-inventory/pkg/connection"
)
```

Reason for this is a rule on the check that imports from `github.com/anchore` are grouped together, FYI this rule is on all go repos at Anchore.
https://github.com/anchore/ecs-inventory/blob/44023bcaca531faccd29af77092b120639ce9823/Makefile#L4